### PR TITLE
IGAPP-342: Fix sharing of PDFs and Images

### PR DESCRIPTION
**Pull request by Taggotty**
This pull request belongs to an issue on our [bugtracker](https://issues.integreat-app.de/).
You can find it there by looking for an issue with the key which is mentioned in the title of this pull request.
It starts with the keyword **IGAPP**.

### CODE REVIEW

File: native/src/modules/layout/components/TransparentHeader.js [hunk 1]
```diff
@@ -1,7 +1,7 @@
 // @flow
 
 import * as React from 'react'
-import Share from 'react-native-share'
+import { Share } from 'react-native'
```
Comment (klinzo) [inline native/src/modules/layout/components/TransparentHeader.js:4]: Can we remove `react-native-share` then?

**Review by klinzo — APPROVED** [review]
Though I don't think this is because of this PR, language change does not change the share url on the dashboard. I.e. if you change language from `de` to `en`, the share url is still https://integreat.app/altmuehlfranken/de`. Could you look into this if this easily fixable? Otherwise I'll just open a new issue.

**Comment by Taggotty** [comment]
> Though I don't think this is because of this PR, language change does not change the share url on the dashboard. I.e. if you change language from `de` to `en`, the share url is still [https://integreat.app/altmuehlfranken/de`](https://integreat.app/altmuehlfranken/de%60). Could you look into this if this easily fixable? Otherwise I'll just open a new issue.

The solution for that should not be that difficult, but I still struggled to make this work today. I'd prefer if you'd create a separate issue for this.

**Review by maxammann — APPROVED** [review]
Tested on Android

**Comment by maxammann** [comment]
@Taggotty @klinzo did anyone of you create the task?

**Pull request merged**